client/scss/components/_block_description.scss

```diff
@@ -0,0 +1,66 @@
+//* @define block-description
+
+.block-description {
+  &:not(.block-description--xlarge) {
+    padding-bottom: 1.2rem;
+    border-bottom: 1px solid color('keyline-grey');
+  }
+}
+
+.block-description__title {
+  margin-top: 0;
+  margin-left: -0.06em;
+
+  .block-description--small & {
```

Comment:
Could we not add these modifiers to the `block-description__title?`
e.g. `block-description__title--medium`?